ha/cli/exec/nodeExecutor.py

```diff
@@ -15,16 +15,63 @@
 # about this software or licensing, please email opensource@seagate.com or
 # cortx-questions@seagate.com.
 
+import argparse
 from ha.core.error import HAUnimplemented
 from ha.cli.exec.commandExecutor import CommandExecutor
+from cortx.utils.log import Log
+
 
 class NodeStartExecutor(CommandExecutor):
 
-    def validate(self) -> str:
-        raise HAUnimplemented("This operation is not implemented.")
-
-    def execute(self) -> str:
-        raise HAUnimplemented("This operation is not implemented.")
+    def __init__(self):
+        """
+        Init cluster start executor
+        """
+        super(NodeStartExecutor, self).__init__()
+        self._args = None
+
+    def validate(self) -> bool:
+        """
+        Validate the command arguments
+        """
+        if self.parse_cluster_args():
+            return True
+        return False
+
+    def parse_cluster_args(self) -> bool:
+        """
+        Parses the command line args.
+        Return: argparse
+        """
+        parser = argparse.ArgumentParser(prog='node start <Node> [all|server] [--json]')
+        parser.add_argument("node", help="Module")
+        parser.add_argument("start", help="action to be performed")
+        parser.add_argument('--nodeid', help='Node to start', required=True)
+        group = parser.add_mutually_exclusive_group(required='True')
+        group.add_argument('--all', action='store_true',
+                           help='All servers to start in a cluster')
```

Comment:
Please fix the help since the 'all' option is for server and storage both.

Reply:
done

Reply:
Please check my latest comment, still need update.